trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -159,7 +159,18 @@ def finalize(self):
             fuel.mass_properties.mass  = 0.
             configuration.fuel         = fuel	
 
-
+        # check if user specifies number of spanwise vortices
```

Comment:
is this the best way to do defaults?

Reply:
currently, yes.  I tried to directly edit the discretization and skip the immediate step but the way AVL was first implemented does not allow it. AVL settings overwrite SUAVE's